docs/source/ccvm_equations_of_motion.rst

```diff
@@ -0,0 +1,44 @@
+
+CCVM Simulator - Equations of Motion
+====================================
+
+Architecture of Coherent Continuous Variable Machine (CCVM)
```

Comment:
Is it not odd not to say "Architecture of Coherent Continuous-Variable Machine (CCVM)" instead of "Architecture of **the** Coherent Continuous-Variable Machine (CCVM)" or "Architecture of Coherent Continuous-Variable Machine**s** (CCVM)", or have you been doing this everywhere? To me, it's like reading "architecture of chair".

Reply:
I asked Farhad. He said 
>This looks correct to me:
>"Architecture of Coherent Continuous-Variable Machines (CCVM)"